shared/constants/types/wallets.js

```diff
@@ -73,7 +73,7 @@ export type _BuildingPayment = {
   currency: string,
   from: string,
   publicMemo: HiddenString,
-  recipientType: ?CounterpartyType,
```

Comment:
seems better to just default to `keybaseUser` (with the blank username, this defaults to showing the search form)